src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/example-global-adf-cross-organization.yml

```
@@ -0,0 +1,341 @@
+              - s3:Put*
+            Resource:
+              - !Sub arn:aws:s3:::${DeploymentAccountBucketName}
+              - !Sub arn:aws:s3:::${DeploymentAccountBucketName}/*
+          - Effect: Allow
+            Sid: "KMS"
+            Action:
+              - kms:Decrypt
+              - kms:DescribeKey
+              - kms:Encrypt
+              - kms:GenerateDataKey*
+              - kms:ReEncrypt*
+            Resource: !Ref KMSArn
+      Roles:
+        - !Ref CloudFormationRole
+  CloudFormationRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: "adf-cloudformation-role"
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS:
+                - !Sub arn:aws:iam::${DeploymentAccountId}:role/adf-codepipeline-role
+                - !Sub arn:aws:iam::${DeploymentAccountId}:role/adf-cloudformation-role
+            Action:
+              - sts:AssumeRole
+      Path: /
+  CloudFormationDeploymentPolicy:
+    # This is the policy that will be used to deploy CloudFormation resources from
+    # within the target account. You should scope this policy depending
+    # On what you would like to deploy within certain Organizational Units.
+    # NOTE: below is a sample IAM policy. This policies should NOT be used for purposes other than testing.
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: "adf-cloudformation-deployment-role-policy-kms"
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Sid: "KMS"
+            Action:
+              - "kms:Decrypt"
+              - "kms:DescribeKey"
+              - "kms:Encrypt"
+              - "kms:GenerateDataKey*"
+              - "kms:ReEncrypt*"
+            Resource: !Ref "KMSArn"
+      Roles:
+        - !Ref CloudFormationDeploymentRole
+  CloudFormationDeploymentRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: "adf-cloudformation-deployment-role"
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Sid: "CloudFormation"
+            Principal:
+              Service:
+                - cloudformation.amazonaws.com
+            Action:
+              - sts:AssumeRole
+          - Effect: Allow
+            Sid: "AssumeRoleLambda"
+            Principal:
+              AWS:
+                - !Sub arn:aws:iam::${DeploymentAccountId}:role/adf-lambda-role
+            Action:
+              - sts:AssumeRole
+          - Effect: Allow
+            Sid: "AssumeRole"
+            Principal:
+              AWS:
+                - !Sub arn:aws:iam::${DeploymentAccountId}:role/adf-codepipeline-role
+            Action:
+              - sts:AssumeRole
+            Condition:
+                ArnEquals:
+                    'aws:SourceArn': !Sub 'arn:aws:codepipeline:${AWS::Region}:${DeploymentAccountId}:*'
```

Indentation and quotes:
```suggestion
              ArnEquals:
                "aws:SourceArn": !Sub "arn:aws:codepipeline:${AWS::Region}:${DeploymentAccountId}:*"
```